tests/testthat/test-epix_fill_through_version.R

```diff
@@ -70,7 +70,7 @@ test_that("epix_fill_through_version does not mutate x", {
     as_epi_archive(tibble::tibble(geo_value = 1L, time_value = 1L, version = 1L, value = 10L))
   )) {
     # We want to perform a strict comparison of the contents of `ea_orig` before
-    # and `ea_orig` after. `clone` + `expect_identical` based on waldo would
+    # and `ea_orig` after. `copy` + `expect_identical` based on waldo would
```

Comment:
Thanks, I didn't look into the comments in this file. I'm noticing a lot of stuff around using `testthat::local_edition(3)`, which wanted to enforce content comparison instead of memory address. I'm thinking we could try removing that too, what do you think?

Reply:
Just tried cleaning a bit of this up and it seems to work (though I'm not fully following where the copies are happening here). Gonna push the commit, but I'll need to remember to come back here and look a little deeper.

Reply:
There's a lot of `testthat::local_edition(3)` to enforce consistency, so if there's another edition-like update, we don't suddenly get different behavior of these tests, and so that theoretically we could piecewise update the tests to the new edition in that case.  We could maybe do that with testthat version pinning or some other mechanism, though AFAIK the DESCRIPTION file doesn't allow for comments that make clear how big a breaking change a testthat version defaulting to a new edition could be.

Reply:
Isn't testthat too embedded in the R ecosystem to make breaking changes? To me, it seems unlikely they'll move away from the behaviors of the current edition (3).

Reply:
It already was with 2 I felt, but they did do 3.

Reply:
"(We don’t anticipate creating new editions very often, and they’ll always be matched with major version, i.e. if there’s another edition, it’ll be the fourth edition and will come with testthat 4.0.0.)" from [here](https://testthat.r-lib.org/articles/third-edition.html) --- they at least keep the possibility open.  And Hadley/RStudio philosophy is different from R Core philosophy, more willing to make breaking changes.

Reply:
Makes sense.  I think having them scattered made more sense for 2 -> 3 so it showed where we actually relied on that behavior, but who knows what would change in 3 -> 4.